server/backend/opendistro_security_plugin.ts

```diff
@@ -17,26 +17,49 @@
 export default function (Client: any, config: any, components: any) {
   const ca = components.clientAction.factory;
 
-  Client.prototype.opendistro_security = components.clientAction.namespaceFactory();
+  if (!Client.prototype.opendistro_security) {
+    Client.prototype.opendistro_security = components.clientAction.namespaceFactory();
+  }
 
+  /**
+   * Gets auth info.
+   */
   Client.prototype.opendistro_security.prototype.authinfo = ca({
     url: {
       fmt: '/_opendistro/_security/authinfo',
     },
   });
 
+  /**
+   * Gets tenant info and kibana server info.
+   *
+   * e.g.
+   * {
+   *   "user_name": "admin",
+   *   "not_fail_on_forbidden_enabled": false,
+   *   "kibana_mt_enabled": true,
+   *   "kibana_index": ".kibana",
+   *   "kibana_server_user": "kibanaserver"
+   * }
+   */
   Client.prototype.opendistro_security.prototype.multitenancyinfo = ca({
     url: {
       fmt: '/_opendistro/_security/kibanainfo',
     },
   });
 
+  /**
+   * Gets tenant info of current user.
```

Comment:
Could you also add response schema like above?

Reply:
added